index.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;500;700&display=swap">
+    <title>Portfolio page </title>
+</head>
+<body>
+    <div class="main-one">
+        <header>
+            <h1 class="mylogo">Ndonyi</h1>
+            <menu>
+                <ul class="home">
+                    <li class="content"><a href="#">Portoflio</a></li>
+                    <li class="content"><a href="https://citec-47.github.io/frontendmasters/">About</a></li>
+                    <li class="content"><a href="https://citec-47.github.io/Desk-version/">Contact</a></li>
+                </ul>
+            </menu>
+          
+             <nav id="statusbar">
+                <div class="parallel"><a href="#"></a></div>
+                <div class="parallel"><a href="#"></a></div>
+                <div class="parallel"><a href="#"></a></div>
+            </nav>
+        </header>
+        
+        <section id="sectionA">
+            <h1 class="introA"><marquee>I’m Ndonyi
+                <br>
+                Glad to see <span>you!</span></marquee></h1>
+                <p class="para-one">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <div class="order">
+                    <h3><a href="#">LET’S CONNECT</a></h3>
+                    <ul>
+                        <li><a href="https://twitter.com/home"><img src="./Images/twiter.jpg"></a></li>
+                        <li><a href="https://www.linkedin.com/feed/"><img src="./Images/linkedin.jpg"></a></li>
+                        <li><a href="#"><img src="./Images/media.jpg"></a></li>
+                        <li><a href="https://github.com/citec-47"><img src="./Images/github.jpg"></li>
+                        <li><a href="https://www.facebook.com/ndonyi.maurice"><img src="./Images/fingers.jpg"></a></li>
+                     </ul>
+                </div>
+            </div>
+</section>
+<section class="cards">
+    <div class="cardern">
+      <img src="./Images/numberone.png" alt="image">
+      <div class="table">
+        <h1>Tonic</h1>
```

Comment:
According to the  [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), you should have just one `h1` for SEO ranking, and also to make your page well structured. [check here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=use%20HTML5%20semantic%20tags).

[Check out this Documentation](https://www.investisdigital.com/blog/seo/using-h1-tag-improves-search-engine-ranking#:~:text=Your%20website%20should,the%20page%20code)